src/OpenTelemetry/Metrics/MeterProviderSdk.cs

```diff
@@ -43,6 +44,7 @@ public class MeterProviderSdk
         {
             this.Resource = resource;
             this.metrics = new Metric[MaxMetrics];
+            this.viewConfig["MyCounter"] = new string[] { "view1", "view2" };
```

Comment:
hardcoded to show idea..

the lookup will need to be based on meter name, version, instrument name, type (or a subset of it).
the value will be name of view, tags_needed,aggregation_overrides_if_any...